src/MicroserviceErrorHandler.js

```diff
@@ -36,8 +36,19 @@ module.exports = class MicroserviceErrorHandler {
 
     createErrorHandler() {
         return (err, req, res, next) => {
-            // eslint-disable-next-line no-param-reassign
-            err.serviceName = this.serviceName;
+            // prevent the service from overriding errors from remote services
+            if (!err.serviceName) {
```

Comment:
Is there a reason you don't user the or operator to assign this value or is it just personal preference?
`err.serviceName = err.serviceName || this.serviceName;`